src/app/workflow/info-tab/info-tab.component.ts

```diff
@@ -150,6 +151,7 @@ export class InfoTabComponent extends EntryTab implements OnInit, OnChanges {
       .subscribe((entryType) => (this.displayTextForButton = this.infoTabService.getTRSId(this.workflow, entryType)));
     this.infoTabService.forumUrlEditing$.pipe(takeUntil(this.ngUnsubscribe)).subscribe((editing) => (this.forumUrlEditing = editing));
     this.infoTabService.topicEditing$.pipe(takeUntil(this.ngUnsubscribe)).subscribe((topicEditing) => (this.topicEditing = topicEditing));
+    this.sourceCodeFile = this.selectedVersion.workflow_path.slice(1); //removes first slash to prevent from displaying on text
```

Comment:
There are several checks for `this.selectedVersion` existing or not, it seems like you should add the check. Also, the other accesses to `selectedVersion` seem to be in `ngOnChanges`; I suggest moving this change there -- if the version changes, `sourceCodeFile` won't get updated.